bootstrap.js

```diff
@@ -53,64 +57,64 @@ function include(src) {
 }
 
 function setDefaultPrefs() {
-  let branch = Services.prefs.getDefaultBranch("");
+  // let branch = Services.prefs.getDefaultBranch('');
   for (let [name, value] in Iterator(DEFAULT_PREFS)) {
     switch (typeof value) {
-    case "boolean":
+    case 'boolean':
       Services.prefs.setBoolPref(name, value);
       break;
-    case "number":
+    case 'number':
       Services.prefs.setIntPref(name, value);
       break;
-    case "string":
+    case 'string':
       Services.prefs.setCharPref(name, value);
       break;
     }
   }
 }
 
 function removeDefaultPrefs() {
-  let branch = Services.prefs.getDefaultBranch("");
-  for (let [name, value] in Iterator(DEFAULT_PREFS)) {
+  let branch = Services.prefs.getDefaultBranch('');
+  for (let [name] in Iterator(DEFAULT_PREFS)) {
     branch.clearUserPref(name);
   }
 }
 
 function install() {
 }
 
-var timer = Cc["@mozilla.org/timer;1"].createInstance(Ci.nsITimer);
+let timer = Cc['@mozilla.org/timer;1'].createInstance(Ci.nsITimer);
 
 function startup(data, reason) {
   // Load helpers from utils.js.
-  include(data.resourceURI.spec + "utils.js");
+  include(data.resourceURI.spec + 'utils.js');
 
   setDefaultPrefs();
 
   // Register the resource:// alias.
-  let resource = Services.io.getProtocolHandler("resource")
+  let resource = Services.io.getProtocolHandler('resource')
                          .QueryInterface(Ci.nsIResProtocolHandler);
   resource.setSubstitution(RESOURCE_HOST, data.resourceURI);
-  unload(function () {
+  unload(function() {
```

Comment:
Huh, so I prefer the space after anonymous functions, because it makes it look less like a function call, and more like we're doing something a little special here…  Can you tweak that rule to allow only this case?